tools/mfassignr/mfassignr_isofiltr.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="mfassignr_isofiltr" name="MFAssignR IsoFiltR: Isotope Filtering" version="@TOOL_VERSION@+galaxy0">
+    <description>
+        IsoFiltR separates likely isotopic masses from monoisotopic masses in a mass list.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+    <expand macro="refs" />
+
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            -e 'source("${mfassignr_isofiltr}")'
+    ]]>
+    </command>
+    <configfiles>
+        <configfile name="mfassignr_isofiltr"><![CDATA[
+            peaks <- read.table("$peaks", header=TRUE, sep="\t")
+            result <- MFAssignR::IsoFiltR(
+                peaks = peaks,
+                SN = $SN,
+                Carbrat = $Carbrat,
+                Sulfrat = $Sulfrat,
+                Sulferr = $Sulferr,
+                Carberr = $Carberr
+            )
+            
+            write.table(result[["Mono"]], file="$mono_out", sep="\t", row.names=FALSE)
+            write.table(result[["Iso"]], file="$iso_out", sep="\t", row.names=FALSE)
+        ]]></configfile>
+    </configfiles>
+    <inputs>
+        <expand macro="isofiltr_param"/>
+    </inputs>
+    <outputs>
+        <data name="mono_out" format="tabular" label="Monoisotopic Masses"/>
+        <data name="iso_out" format="tabular" label="Isotopic Masses"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="peaks" value="QC1_1_POS_500.tabular" ftype="tabular"/>
+            <output name="mono_out" file="isofiltr_output1.tabular" ftype="tabular"/>
+            <output name="iso_out" file="isofiltr_output2.tabular" ftype="tabular"/>
+        </test>
+    </tests>
+    <help>
+        IsoFiltR identifies and separates likely isotopic masses from monoisotopic masses in a mass list. 
+        This should be done prior to formula assignment to reduce incorrect formula assignments.
```

Comment:
Please include information about the expected format of the input table - ideally an example in rst so that it is rendered here as a table would be nice.

Reply:
done in [5d269af](https://github.com/RECETOX/galaxytools/pull/565/commits/5d269afce6a1cdba26dc9813a12a9c5bffa94756)